recipes/asyncio-atexit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "asyncio-atexit" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asyncio-atexit-{{ version }}.tar.gz
+  sha256: 1d0c71544b8ee2c484d322844ee72c0875dde6f250c0ed5b6993592ab9f7d436
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - asyncio_atexit
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/minrk/asyncio-atexit
+  summary: Like atexit, but for asyncio
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jacobtomlinson
```

Comment:
@minrk I see this package isn't on conda forge and I need it for my package. Are you happy with me adding this and are you ok with me adding you to this list?

Reply:
I've moved this into a separate PR https://github.com/conda-forge/staged-recipes/pull/23181